sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -601,6 +617,19 @@ public partial class AvailabilityZoneMapping
         public string AvailabilityZone { get { throw null; } }
         public bool? IsAvailable { get { throw null; } }
     }
+    public partial class BackupRestoreFiles
+    {
+        public BackupRestoreFiles(System.Collections.Generic.IEnumerable<string> fileList, string destinationVolumeId) { }
+        public string DestinationVolumeId { get { throw null; } }
+        public System.Collections.Generic.IList<string> FileList { get { throw null; } }
+        public string RestoreFilePath { get { throw null; } set { } }
+    }
+    public partial class BreakFileLocksContent
```

Comment:
Could we prepend the name of the specific resource name that using this to specify it? It should be `NetAppVolumeBreakFileLocksContent`

Reply:
This is not a resource it is a body payload to an `POST` action, should we still rename it?